Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -674,11 +674,11 @@ WarpXParticleContainer::GetChargeDensity (int lev, bool local)
     auto rho = std::make_unique<MultiFab>(nba,dm,WarpX::ncomps,ng_rho);
     rho->setVal(0.0);
 
-#ifdef _OPENMP
+#ifdef AMREX_USE_OMP
 #pragma omp parallel
```

Comment:
Missing `if (amrex::Gpu::notInLaunchRegion())`?

Reply:
I agree with you that `if (amrex::Gpu::notInLaunchRegion())` should be there.

Reply:
```suggestion
#pragma omp parallel if (amrex::Gpu::notInLaunchRegion())
```